eng/RepoLayout.props

```
@@ -9,8 +9,8 @@
     <!-- Set version for runtimeconfig.json, used by DotnetRuntimeBootstrapper for minimal version to support
          To be kept in sync with the .NET SDK version in global.json
     -->
-    <RuntimeFrameworkVersion>8.0.0</RuntimeFrameworkVersion>
-    <SolutionTargetFramework>net8.0-windows</SolutionTargetFramework>
+    <RuntimeFrameworkVersion>9.0.0</RuntimeFrameworkVersion>
```

❔ less strict in order to avoid build broken by MSVS updating the SDK
```suggestion
    <RuntimeFrameworkVersion>9.0</RuntimeFrameworkVersion>
```